packages/ts-autocomplete/src/index.ts

```diff
@@ -165,10 +165,7 @@ function filterDiagnostics(diagnostics: ts.Diagnostic[]): {
       ..._.pick(item, 'messageText'),
     };
 
-    if (
-      result.fileName === '/shell-api.ts' ||
-      result.fileName?.startsWith('/connection-')
-    ) {
+    if (result.fileName?.endsWith('shell-api.ts')) {
```

Comment:
connection files actually start with connection_, not connection- but it is only the -shell-api.ts ones that are huge and clog up debug output, causing it to be cut off and then we can't see all the actual errors.